partiql-parser/src/lalr/partiql.lalrpop

```diff
@@ -0,0 +1,584 @@
+use std::str::FromStr;
```

Comment:
FYI for other reviewers. CLion has a [LALRPOP plugin](https://plugins.jetbrains.com/plugin/15229-lalrpop) which can make this file easier to view